docs/pages/docs/quint.md

```diff
@@ -85,12 +85,14 @@ Options:
   --temporal   the temporal properties to check, separated by commas    [string]
   --target     the compilation target. Supported values: tlaplus, json
                                                       [string] [default: "json"]
+  --flatten           Whether or not to flatten the modules into one
```

Comment:
Since the flag is on by default, we want might to show how disable it explicitly in its help message:

```suggestion
  --flatten=[true|false]           Whether or not to flatten the modules into one
```

I know it is also explained below, but it can't hurt to have it where the flag is defined as well.

Reply:
I couldn't make `yargs` print the `--help` text like that, but did this instead:

```
  --flatten           Whether or not to flatten the modules into one. Use
                      --flatten=false to disable       [boolean] [default: true]
```

Also, I realize we could use `choices` for stuff like `--backend` and `--target`, so I changed that and removed our manual checks. Error messages now look like this:
```
$ quint compile --target tla file.qnt
[...]
Invalid values:
  Argument: target, Given: "tla", Choices: "tlaplus", "json"
```

I didn't used `choices` for `flatten` cause it would print something silly like `[boolean] [choices: true, false]`